core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -357,9 +357,6 @@ private DefaultClientRequestContext(DefaultClientRequestContext ctx,
         if (ctx.request() != null) {
             requireNonNull(req, "req");
         }
-        if (ctx.rpcRequest() != null) {
-            requireNonNull(rpcReq, "rpcReq");
-        }
```

Comment:
How about adding some comment about why we don't check nullness of `rpcReq` here unlike `req`?

Reply:
Added. 😄

Reply:
And please add a link to the related github issue and PR? 😉

Reply:
Added again. 😄